TVC_MVP/CountyViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  CountyTableViewController.swift
+//  TVC_MVP
+//
+//  Created by Shane McCully on 28/02/2019.
+//  Copyright © 2019 Shane McCully. All rights reserved.
+//
+
+import UIKit
+
+private struct Constants {
+
+    static let cellIdentifier = "CountyCell"
+    static let okayString = "Okay"
+    static let removeFromList = "Remove From List"
+    static let errorText = "fatal error : fetchCounty"
+
+}
+
+class CountyViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, CountyViewProtocol {
+
+    var presenter: CountyPresnterProtocol!
+    @IBOutlet weak var tableView: UITableView!
```

Comment:
Missing `private` and we always put the `IBOutlet` on top of the other properties